pom.xml

```diff
@@ -35,6 +35,51 @@
 				</exclusion>
 			</exclusions>
 		</dependency>
+		<dependency>
+			<groupId>com.example</groupId>
+			<artifactId>spring-boot</artifactId>
+			<version>0.0.1-SNAPSHOT</version>
+			<scope>test</scope>
+		</dependency>
+
+		<!-- https://mvnrepository.com/artifact/org.testng/testng -->
+		<dependency>
+			<groupId>org.testng</groupId>
+			<artifactId>testng</artifactId>
+			<version>6.14.3</version>
```

Comment:
it is better to move versions to props 

![image](https://user-images.githubusercontent.com/3214148/104086363-86ac4000-5257-11eb-9832-0d08217dccab.png)


https://stackoverflow.com/questions/19123013/maven-version-with-a-property